aries-site/src/examples/components/input/TextInputExample.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Box, FormField } from 'grommet';
+import { TextInput } from 'aries-core';
+
+import { FormRow, UsageExample } from '../../../layouts';
+
+const usernamePlaceholder = 'Email address';
+const passwordPlaceholder = 'Password';
+
+const LabelInlineExample = () => {
```

Comment:
This example is having a very difficult time in that in between medium/small screen where the main content width has already hit small. The username starts to overlap the textinput heavily. I'm wondering how we can improve this experience if possible.

Reply:
Agree. Thinking we separate into two UsageExample instances, but was trying to stay true to design mock.

Reply:
Removed left labels per earlier discussion